app/utils/validation.js

```diff
@@ -48,16 +48,21 @@ export const sameAs = (otherField, message) => (value, context) => [
   message,
 ];
 
-export function createValidator(fieldValidators) {
+export function createValidator(fieldValidators, rawValidator) {
+
   return function validate(input) {
-    return Object.keys(fieldValidators).reduce((errors, field) => {
+  const resultRawValidator = rawValidator(input)
+
+    const validatorErrors = Object.keys(fieldValidators).reduce((errors, field) => {
       const fieldErrors =
         fieldValidators[field]
           .map((validator) => validator(input[field], input))
           .filter(([isValid]) => !isValid)
           .map(([, message]) => message || 'not valid') || 0;
       if (fieldErrors.length) errors[field] = fieldErrors;
       return errors;
-    }, {});
+    }, 
+    {});
+    return merge(validatorErrors, resultRawValidator)
```

Comment:
...then when your merge them, the name's makes it easy to see that the input parameters are used, then merged, into errors